README.md

```diff
@@ -3,10 +3,11 @@
 GDPR SDK for integrating Go services with AGS (AccelByte Gaming Services) GDPR service.
 
 This GDPR SDK could be used by participant services to integrate into AGS GDPR workflow.
-There are 3 GDPR workflow that this GDPR SDK supported:
+There are 4 GDPR workflow that this GDPR SDK supported:
 1. Right to data portability
 2. Right to erasure (right to be forgotten)
 3. Right to restrict processing
+4. Right to erase 3rd party account information
```

Comment:
Please add the example in the README.md as well, 
please follow the example of the other workflow